Detectors/TRD/workflow/include/TRDWorkflow/VdAndExBCalibSpec.h

```diff
@@ -47,7 +50,16 @@ class VdAndExBCalibDevice : public o2::framework::Task
     auto enableOutput = ic.options().get<bool>("enable-root-output");
     auto slotL = ic.options().get<uint32_t>("sec-per-slot");
     auto delay = ic.options().get<uint32_t>("max-delay");
-    mCalibrator = std::make_unique<o2::trd::CalibratorVdExB>(enableOutput);
+    if (enableOutput) {
```

Comment:
I would prefer to move this block completely into `CalibratorVdExB.cxx`. And here inside the spec you could just call a function e.g. `mCalibrator->createOutputFile()` in case ROOT output is enabled. Then you could also leave the default constructor as it is now.
Inside the `CalibratorVdExB` you could then also switch to smart pointers which are preferred: `mOutFile = std::make_unique<TFile>("filename", "recreate")`